code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -349,3 +349,16 @@
 	sword.worn_icon_state = "e_sword_on_red"
 
 	H.update_inv_hands()
+
+
+/datum/antagonist/traitor/get_preview_icon()
+	var/mob/living/carbon/human/dummy/consistent/traitor_dummy = new
+	traitor_dummy.hair_color = "#dba86e53"
+	traitor_dummy.hairstyle = "CIA"
+	traitor_dummy.update_hair(TRUE)
```

Comment:
I'm not sure about the 'CIA' hairstyle on this one. Combined with the gas mask it looks a bit like some kind of weird hat:
![image](https://user-images.githubusercontent.com/57483089/175554519-cdec0da8-7bb8-4f8b-9127-3715d19f6a6b.png)

Reply:
Changed to business hair and made grey-ish black.

Reply:
![image](https://user-images.githubusercontent.com/9637690/175785379-4c1d9e88-5246-4eb9-ad5b-0725908792ff.png)